setup.py

```diff
@@ -14,7 +14,8 @@ def read(f):
 ]
```

Comment:
Any need to upgrade this library? `mistune`

Reply:
we are using v1 of this library: https://github.com/lepture/mistune/tree/v1 0.8.4 is last version of v1 then there is v2 and v3 which may break compatibility - I've have not checked but:

- v1 docs says it's python 3.5+ compatible
- we have tests covering usage of mistune starting from `ParserTestCase` and it seems coverage is pretty good.